addons/mass_mailing_sms/models/mailing_mailing.py

```diff
@@ -314,12 +314,12 @@ def convert_links(self):
 
     def get_sms_link_replacements_placeholders(self):
         """Get placeholders for replaced links in sms widget for accurate computation of sms counts.
-
         Reminders and assumptions:
-          * Links wille be transformed to the format "[base_url]/r/[link_tracker_code]/s/[sms_id]".
-          * unsubscribe is formatted as: "\nSTOP SMS : [base_url]/sms/[mailing_id]/[trace_code]".
 
-        :return: Character counts used for links, formatted as `{link: str, unsubscribe: str}`.
+        * Links wille be transformed to the format "[base_url]/r/[link_tracker_code]/s/[sms_id]".
+        * unsubscribe is formatted as: "STOP SMS : [base_url]/sms/[mailing_id]/[trace_code]".
+
+        :returns: Character counts used for links, formatted as `{link: str, unsubscribe: str}`.
```

Comment:
```suggestion
        :returns: Character counts used for links, formatted as ``{link: str, unsubscribe: str}``.
```
Single backticks produce `<title_reference>` but we want a `<literal>` here: double-backticks.
```xml
            <field_name>
                return
            <field_body>
                <paragraph>
                    Character counts used for links, formatted as 
                    <title_reference>
                        {link: str, unsubscribe: str}
                    .
```